### PR TITLE
fix: anonymous not use token, can not paste otp

### PR DESCRIPTION
**Pull request by nguyenhoaidanh**

### CODE REVIEW

File: src/hooks/useKyberSwapConfig.ts [hunk 1]
```diff
@@ -66,7 +66,7 @@ const parseGlobalResponse = (
 ): KyberswapGlobalConfig => {
   const data = responseData?.data?.config
   const aggregatorDomain = data?.aggregator ?? AGGREGATOR_API
-  const isEnableAuthenAggregator = !!data?.isEnableAuthenAggregator
+  const isEnableAuthenAggregator = !data ? true : !!data?.isEnableAuthenAggregator
```
Comment (nhd98z) [inline src/hooks/useKyberSwapConfig.ts:69]: What's this for?

**Review by nhd98z — CHANGES_REQUESTED** [review]

File: src/pages/Verify/VerifyCodeModal/OtpInput.tsx [hunk 2]
```diff
@@ -171,6 +172,8 @@ const OTPInput = ({
       .slice(0, numInputs - activeInput)
       .split('')
 
+    if (isInputNum && pastedData.some(el => isNaN(+el))) return
```
Comment (namgold) [inline src/pages/Verify/VerifyCodeModal/OtpInput.tsx:175]:
```suggestion
    if (isInputNum && pastedData.some(el => isNaN(el))) return
```
this is dont needed

File: src/hooks/useKyberSwapConfig.ts (hunk 1 above)
Comment (viet-nv) [reply src/hooks/useKyberSwapConfig.ts:69]: for anonymous user?

Comment (nguyenhoaidanh) [reply src/hooks/useKyberSwapConfig.ts:69]: this config decide use access token or not.
update to true in default

File: src/pages/Verify/VerifyCodeModal/OtpInput.tsx (hunk 2 above)
Comment (nguyenhoaidanh) [reply src/pages/Verify/VerifyCodeModal/OtpInput.tsx:175]: no, it is string

File: src/hooks/useKyberSwapConfig.ts (hunk 1 above)
Comment (nguyenhoaidanh) [reply src/hooks/useKyberSwapConfig.ts:69]: yep. or when fetching

File: src/pages/Verify/VerifyCodeModal/OtpInput.tsx (hunk 2 above)
Comment (nguyenhoaidanh) [reply src/pages/Verify/VerifyCodeModal/OtpInput.tsx:175]: but typescript err =)) so I will keep it

File: src/hooks/useKyberSwapConfig.ts (hunk 1 above)
Comment (namgold) [reply src/hooks/useKyberSwapConfig.ts:69]: should we create a const that `DEFAULT_is_Enable_Authen_Aggregator = true`?

File: src/pages/Verify/VerifyCodeModal/OtpInput.tsx (hunk 2 above)
Comment (namgold) [reply src/pages/Verify/VerifyCodeModal/OtpInput.tsx:175]: `as any`

Comment (nguyenhoaidanh) [reply src/pages/Verify/VerifyCodeModal/OtpInput.tsx:175]: 🙏

**Review by nhd98z — APPROVED** [review]

**Pull request merged**